pkg/upgrade/docker_utils_test.go

```diff
@@ -96,91 +89,6 @@ func constructVariables(t *testing.T) map[string]string {
 	return envVars
 }
 
-func streamDockerLogs(containerName string) (chan string, func(), error) {
-	logsCmd := exec.Command("docker", "logs", "-f", containerName)
-	stdoutPipe, err := logsCmd.StdoutPipe()
-	if err != nil {
-		return nil, nil, err
-	}
-
-	err = logsCmd.Start()
-	if err != nil {
-		return nil, nil, err
-	}
-
-	logChan := make(chan string)
-	go func() {
-		scanner := bufio.NewScanner(stdoutPipe)
-		for scanner.Scan() {
-			logChan <- scanner.Text()
-		}
-		close(logChan)
-	}()
-
-	cancelFunc := func() {
-		_ = logsCmd.Process.Kill()
-	}
-
-	return logChan, cancelFunc, nil
-}
-
-func runContainer(
-	t *testing.T,
-	containerName string,
-	imageName string,
-	envVars map[string]string,
-) {
-	var dockerEnvArgs []string
-	for key, value := range envVars {
-		dockerEnvArgs = append(dockerEnvArgs, "-e", fmt.Sprintf("%s=%s", key, value))
-	}
-
-	_ = dockerRmc(containerName)
-
-	dockerCmd := []string{"run", "-d"}
-	if runtime.GOOS == "linux" {
-		dockerCmd = append(dockerCmd, "--add-host=host.docker.internal:host-gateway")
```

Comment:
you need this line

Reply:
Yeah, realized that when it ran in an ubuntu container. Added into as part of the ContainerRequest.